inc/vcf/error.hpp

```diff
@@ -290,7 +301,10 @@ namespace ebi
                       ErrorFix error_fix = ErrorFix::IRRECOVERABLE_VALUE,
                       const std::string &field = "",
                       const std::string &expected_value = "")
-                : BodySectionError{line, message}, error_fix{error_fix}, field{field}, expected_value{expected_value} {
+                : BodySectionError{line, message, detailed_message},
+                error_fix{error_fix},
+                field{field},
+                expected_value{expected_value} {
                     if (error_fix == ErrorFix::RECOVERABLE_VALUE && expected_value.empty()) {
                         throw std::invalid_argument{"The program had an internal error: An error with recoverable value must provide a non-empty expected value"};
```

Comment:
This message seems to be as critical as the one above, but only this one makes it clear enough. I would suggest rephrasing:

> The program had an internal error

as 

> An internal error has occurred

and displaying it in both error classes.